game.rb

```diff
@@ -3,15 +3,30 @@
 
 class Game
 
-  def initialize io_handler, prompter
+  attr_reader :word, :letter
+
+  def initialize game_view, io_handler, prompter, validator
+    @game_view = game_view
     @io_handler = io_handler
     @prompter = prompter
+    @validator = validator
   end
 
   def game_handler
     @io_handler.printer(@prompter.game_greeting)
     @io_handler.printer(@prompter.prompt_for_word)
-    @io_handler.printer(@io_handler.word_blank(@io_handler.get_input))
+    while @validator.is_a_word?(@word=(@io_handler.get_input.upcase)) == false
```

Comment:
This functionality does not belong in your game loop. Your game loop should not be validating word choice, it should only take a valid word and play the game.